Makefile

```diff
@@ -75,6 +75,10 @@ debug-all:
 	mkdir -p $(builddir)/debug
 	cd $(builddir)/debug && cmake -D BUILD_TESTS=ON -D BUILD_SHARED_LIBS=OFF -D CMAKE_BUILD_TYPE=Debug $(topdir) && $(MAKE)
 
+debug-all-icecc:
+	mkdir -p $(builddir)/debug-icecc
```

Comment:
Should it be debug-icecc or just debug as with all other targets? Note - icecc compiles a generic (not native) binary.

Reply:
I understand your rationale, but I don't think it holds: "native" itself depends on cmake settings (machine architecture), so it's highly variable, just like debug (in fact, moreso, because for icecc debug, you end up building a universal binary). If there were a per-config cache, then what you're saying might make sense, but that doesn't exist, and would be a rather atypical make setup (delete all products and rebuild, or set a custom output dir, tends to be the appropriate strategy with make).

Reply:
Ok, thanks.
I'm just being extra careful with changing the default behavior/assumptions.

Reply:
Now building in standard locations.